test/functional/update/update-minversion.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10
+	create_bundle -L -n bundle1 -v 10 -f /file1 "$TEST_NAME"
+
+	create_version "$TEST_NAME" 20 10 staging
+	update_minversion 20
+}
+
+@test "skip fullfile download for unchanged file in minversion update " {
+
+	run sudo sh -c "$SWUPD update $SWUPD_OPTS"
+	assert_status_is 0
+
+	expected_output=$(cat <<-EOM
```

Comment:
This test is verifying that the file wasn't downloaded just by checking the output of swupd. Can you also *remove* the fullfile from the webdir before performing the update so the update will fail if it actually tries to download the file.

What sometimes happens when swupd output changes is that the developer comes and blanket fixes all the failing output tests without verifying if it should be updated or not. By removing the fallback we can make sure the test will always fail if it does the wrong thing.

Reply:
Ok, I'll clean the files directory.

Reply:
done in new revision